internal/format_test.ts

```diff
@@ -21,46 +21,47 @@ ${red("-   d")}
   );
 });
 
-// Check that the diff formatter overrides some default behaviours of
-// `Deno.inspect()` which are problematic for diffing.
-Deno.test("format() overrides default behaviours of Deno.inspect", async (t) => {
-  // Wraps objects into multiple lines even when they are small. Prints trailing
-  // commas.
-  await t.step(
-    "format() always wraps objects into multiple lines and prints trailing commas",
-    () =>
-      assertEquals(
-        stripAnsiCode(format({ a: 1, b: 2 })),
-        `{
+if (typeof Deno.inspect === "function") {
```

Comment:
Can you use `ignore` option of `Deno.test` instead of big `if` block to exclude some tests in some environment? ref https://docs.deno.com/api/deno/~/Deno.TestDefinition